pkg/features/kube_features.go

```diff
@@ -138,6 +138,14 @@ const (
 	// Postpone deletion of a PV or a PVC when they are being used
 	StorageObjectInUseProtection featuregate.Feature = "StorageObjectInUseProtection"
 
+	// owner: @saad-ali
+	// ga: 	  v1.10
+	//
+	// Allow mounting a subpath of a volume in a container
+	// NOTE: This feature gate has been deprecated and is no longer enforced.
+	// It will be completely removed in 1.24. Until then, it's still visible in `kubelet --help`
```

Comment:
According to the deprecation policy, we need to keep the feature gate for 2 releases: https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecation

Reply:
noted, I'll update the removal release to 1.25